src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -25,16 +25,13 @@ namespace Tizen.NUI.Components
     /// <summary>
     /// [Draft] This class provides a View that can layouting items in list and grid with high performance.
     /// </summary>
-    /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
     [EditorBrowsable(EditorBrowsableState.Never)]
     public abstract class ItemsView : ScrollableBase, ICollectionChangedNotifier
```

Comment:
For now, it seems that ICollectionChangedNotifier is not required in ItemsView class.

Reply:
I commented out for now, but I think it need here either.
will be refreshed when notify feature is implemented.